storage/innobase/srv/srv0srv.cc

```diff
@@ -1111,7 +1111,7 @@ srv_export_innodb_status(void)
 
 	export_vars.innodb_log_writes = srv_stats.log_writes;
 
-	mysql_mutex_lock(&lock_sys.wait_mutex);
+	lock_sys.wait_mutex_lock();
```

Comment:
This code is only executed during `SHOW` commands that would access `innodb_status_variables_export`. I do not think that spinning would make any observable improvement here.